packages/cozy-client/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@cozy/minilog": "1.0.0",
+    "@redux-devtools/extension": "^3.2.2",
```

Comment:
this is a bit annoying to make all the consuming apps installing @redux-devtools/extension for one debug case, in my opinion.

Reply:
it's 200 lines of code... But yeah I think that just a `window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()` is enough